op-e2e/setup.go

```diff
@@ -380,7 +381,7 @@ func (cfg SystemConfig) start() (*System, error) {
 	sys.cfg.RollupConfig.Genesis = sys.RolupGenesis
 	sys.cfg.RollupConfig.BatchSenderAddress = batchSubmitterAddr
 	sys.cfg.RollupConfig.P2PSequencerAddress = p2pSignerAddr
-	sys.cfg.L2OOCfg.L2StartTime = new(big.Int).SetUint64(l2GenesisTime)
+	sys.cfg.L2OOCfg.L2StartingBlock = new(big.Int)
```

Comment:
@trianglesphere I think the following would be more accurate, and I'd like to add a check to the constructor to enforce `_historicalTotalBlocks > _startingBlockNumber`. 

```suggestion
	sys.cfg.L2OOCfg.L2StartingBlock = new(big.Int).SetInt64(int64(l2GenesisID.Number))
```

Reply:
Ok, let me update this to make a little more sense with the golang types, but we can go ahead with this.